src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -100,34 +209,165 @@ const SelectableAssetItem = ({tokenInfo, disabled, wallet}: SelectableAssetItemP
   }
 
   return (
-    <TouchableOpacity style={styles.item} onPress={onSelect} testID="selectTokenButton" disabled={disabled}>
+    <TouchableOpacity
+      style={[styles.item, isPrimary && styles.borderBottom]}
+      onPress={onSelect}
+      testID="selectTokenButton"
+      disabled={disabled}
+    >
       <AmountItem amount={{tokenId: tokenInfo.id, quantity: spendable}} wallet={wallet} />
     </TouchableOpacity>
   )
 }
 
+const ListEmptyComponent = ({fungibilityFilter}: {fungibilityFilter: FungibilityFilter}) => {
+  const {search: assetSearchTerm, visible: isSearching} = useSearch()
+  const wallet = useSelectedWallet()
+  const {nfts} = useNfts(wallet)
+  const filteredTokenInfos = useFilteredTokenInfos({fungibilityFilter})
+  const strings = useStrings()
+
+  const isWalletEmpty = useIsWalletEmpty()
+
+  if (isSearching && assetSearchTerm.length > 0 && filteredTokenInfos.length === 0) return <EmptySearchResult />
+
+  if (fungibilityFilter === 'ft' && isWalletEmpty && !isSearching)
+    return <NoAssetsYet text={strings.noAssetsAddedYet(strings.tokens(2))} />
+
+  if (fungibilityFilter === 'nft' && nfts.length === 0)
+    return <NoAssetsYet text={strings.noAssetsAddedYet(strings.nfts(2))} />
+
+  return null
+}
+
+const NoAssetsYet = ({text}: {text: string}) => {
+  return (
+    <View style={styles.imageContainer}>
+      <Spacer height={160} />
+
+      <NoAssetFounfImage style={styles.image} />
+
+      <Spacer height={25} />
+
+      <Text style={styles.contentText}>{text}</Text>
+    </View>
+  )
+}
+
+const EmptySearchResult = () => {
+  const strings = useStrings()
+  return (
+    <View style={styles.imageContainer}>
+      <Spacer height={160} />
+
+      <NoAssetFounfImage style={styles.image} />
+
+      <Spacer height={25} />
+
+      <Text style={styles.contentText}>{strings.noAssets}</Text>
+    </View>
+  )
+}
+
+const useIsWalletEmpty = () => {
+  const wallet = useSelectedWallet()
+  const balances = useBalances(wallet)
+
+  const tokenInfos = useTokenInfos({
+    wallet,
+    tokenIds: Amounts.toArray(balances).map(({tokenId}) => tokenId),
+  })
+
+  return (
+    tokenInfos.length === 1 &&
+    tokenInfos[0].id === wallet.primaryTokenInfo.id &&
+    Amounts.getAmount(balances, wallet.primaryTokenInfo.id).quantity === Quantities.zero
+  )
+}
```

Comment:
I think this will make it cleaner.
```suggestion
  const balances = useBalances(wallet)
  return Amounts.toArray(balances).every(({quantity}) => quantity === Quantities.zero)
}
```

Reply:
Quantities.isZero(quantity)